### PR TITLE
unify pages to "public chains" page; add Nova info

### PR DESCRIPTION
**Pull request by DZGoldman**
https://deploy-preview-52--hopeful-rosalind-235059.netlify.app/docs/public_chains
https://deploy-preview-52--hopeful-rosalind-235059.netlify.app/docs/useful_addresses

(from https://github.com/OffchainLabs/developer-website/pull/52)

### CODE REVIEW

File: docs/Public_Chains.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+#### Get Some Native Currency
+
+You'll need a chain's native currency to transact. You can either acquire funds directly on an Arbitrum chain, or get funds on a chain's underlying L1 and bridge it across. You can get testnet Ether from the following faucets:
+
+- [Goerli](https://goerlifaucet.com/)
+- [Rinkeby](https://faucet.rinkeby.io/)
+- [Nitro Goerli Rollup](https://twitter.com/intent/tweet?text=ok%20I%20need%20@arbitrum%20to%20give%20me%20Nitro%20testnet%20gas.%20like%20VERY%20SOON.%20I%20cant%20take%20this,%20I%E2%80%99ve%20been%20waiting%20for%20@nitro_devnet%20release.%20I%20just%20want%20to%20start%20developing.%20but%20I%20need%20the%20gas%20IN%20MY%20WALLET%20NOW.%20can%20devs%20DO%20SOMETHING??%20%20SEND%20HERE:%200xAddA0B73Fe69a6E3e7c1072Bb9523105753e08f8)
+
+[Supported centralized exchanges](https://portal.arbitrum.one/#centralizedexchanges) allow you purchase (mainnet) Ether and withdraw it directly onto Arbitrum one.
```
Comment (pearring) [inline docs/Public_Chains.md:49]:
```suggestion
[Supported centralized exchanges](https://portal.arbitrum.one/#centralizedexchanges) allow you to purchase (mainnet) Ether and withdraw it directly onto Arbitrum one.
```

File: docs/Public_Chains.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Use L2 Dapps!
+
+Interacting with Arbitrum chains will feel very similar to using Ethereum, just cheaper and faster! To get a sense of what's out there, you can check out our [portal page](https://portal.arbitrum.one/), where we showcase some of the dApps, wallets, and infrastructure currently live on Arbitrum One.
+
+### Build on Arbitrum
+
+Dapp developers can build on Arbitrum seamlessly using their favorite Ethereum tooling; see [here](Contract_Deployment.md) for contract deployment and [here](Frontend_Integration.md) for frontend integration.
+
+### What Next
```
Comment (pearring) [inline docs/Public_Chains.md:63]:
```suggestion
### What's Next
```

File: docs/Public_Chains.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+| ---------------------------------------- | -------- | ------- | ------------- | ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ | --------------- | -------------------------------------------- |
+| Arbitrum One                             | 42161    | Mainnet | Ethereum      | Classic Rollup     | https://arb1.arbitrum.io/rpc <br> https://arbitrum-mainnet.infura.io/v3/YOUR-PROJECT-ID <br> https://arb-mainnet.g.alchemy.com/v2/your-api-key | https://arbiscan.io/ https://explorer.arbitrum.io/                 | ETH             | https://retryable-dashboard.arbitrum.io/     |
+| Arbitrum Nova                            | 42170    | Mainnet | Ethereum      | Nitro AnyTrust     | [coming soon!]                                                                                                                                 | https://a4ba-explorer.arbitrum.io/                                 | ETH             | http://retryable-tx-panel-nitro.arbitrum.io/ |
```
Comment (pearring) [inline docs/Public_Chains.md]: I think we're ok to start using the new tld names:
Explorer: https://nova-explorer.arbitrum.io/
RPC URL: https://nova.arbitrum.io/rpc <-- we're already sending this to teams so it should be ok to share publicly?

Comment (pearring) [reply docs/Public_Chains.md]: for some reason it wouldn't let me issue a suggestion here

File: docs/Public_Chains.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+You'll need a chain's native currency to transact. You can either acquire funds directly on an Arbitrum chain, or get funds on a chain's underlying L1 and bridge it across. You can get testnet Ether from the following faucets: 
+- [Goerli](https://goerlifaucet.com/)
+- [Rinkeby](https://faucet.rinkeby.io/)
+- [Nitro Goerli Rollup](https://twitter.com/intent/tweet?text=ok%20I%20need%20@arbitrum%20to%20give%20me%20Nitro%20testnet%20gas.%20like%20VERY%20SOON.%20I%20cant%20take%20this,%20I%E2%80%99ve%20been%20waiting%20for%20@nitro_devnet%20release.%20I%20just%20want%20to%20start%20developing.%20but%20I%20need%20the%20gas%20IN%20MY%20WALLET%20NOW.%20can%20devs%20DO%20SOMETHING??%20%20SEND%20HERE:%200xAddA0B73Fe69a6E3e7c1072Bb9523105753e08f8)
+
+[Supported centralized exchanges](https://portal.arbitrum.one/#centralizedexchanges) allow you purchase (mainnet) Ether and withdraw it directly onto Arbitrum one.
+
+
+### Deposit And Withdraw
+To move your Ether and Tokens between Arbitrum and Ethereum chains at [bridge.arbitrum.io](https://bridge.arbitrum.io/).
```
Comment (pearring) [inline docs/Public_Chains.md:53]: ```To move your Ether and Tokens between Arbitrum and Ethereum chains, visit[bridge.arbitrum.io](https://bridge.arbitrum.io/).```

**Review by pearring — CHANGES_REQUESTED** [review]
LGTM pending a few changes I added.

**Comment by DZGoldman** [comment]
@pearring updated (preview build shows the latest)

**Review by pearring — APPROVED** [review]
LGTM

**Pull request merged**